spec

```diff
@@ -4223,8 +4223,6 @@ beginseg
     name "ovl_En_Ot"
     compress
     include "build/src/overlays/actors/ovl_En_Ot/z_en_ot.o"
-    include "build/data/ovl_En_Ot/ovl_En_Ot.data.o"
-    include "build/data/ovl_En_Ot/ovl_En_Ot.bss.o"
     include "build/data/ovl_En_Ot/ovl_En_Ot.reloc.o"
```

Comment:
You need to `#ifdef NON_MATCHING` and add the generated reloc if you have nonmatchings in a file. See https://hackmd.io/z_yZ-9y6R56pF_9PiEQ7fA?both#linker_scriptscode_scripttxt-is-gone-replaced-by-spec for the format.